amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -3237,11 +3253,10 @@ type Expenditure @model {
   isStaked: Boolean!
   type: ExpenditureType!
   """
-  Hash of the `ExpenditureAdded` event transaction
-  @TODO: Make this a non-nullable field once existing data is updated
+  Actions associated with the expenditure
   """
-  transactionHash: ID
-  action: ColonyAction @hasOne(fields: ["transactionHash"])
+  actions: [ColonyAction!]
+    @hasMany(indexName: "byExpenditureId", fields: ["id"])
```

Comment:
I have no idea how we need to handle this migration on the db side. But there's a good chance it's done automatically for us.

Reply:
Fortunately it's never been deployed. The original `action` field was only added in #2015.